src/GBLesson1/MyArrayDataException.java

```diff
@@ -0,0 +1,9 @@
+package GBLesson1;
+
+public class MyArrayDataException extends RuntimeException{
+
+    public MyArrayDataException(int i, int j){
+        System.out.println( "Элемент c индексом" + i + j + "Нельзя преобразовать в число");
```

Comment:
```suggestion
        super( "Элемент c индексом" + i + j + "Нельзя преобразовать в число");
```
конструктор должен служить только для инициализации состояния, следует избегать в нем любой сторонней логики